objectRecognitionSource/README.md

```diff
@@ -450,12 +471,45 @@ The options are as follows. Remember to prepend "NN" when using an option in a Q
 
 No additional data is sent.
 
+#### <a name="modelFiles" id="modelFiles"></a>YOLO Model File(s) used in the Tests 
+
+Generally speaking, the model files are large.  The protocol buffer ('.pb') file used in the tests is around 200 megabytes.  Consequently, we download these as part of the build, but they are not stored as part of the Git project.
+
+The files we use are as follows.
+
+* COCO -- COCO is a large dataset from Microsoft with 80 object categories.
+  * Version 1.1 -- the primary difference over version 1.1 is the providing of the `.meta` file. However, the file has been regenerated so we view it as a new version.
+     * `coco-1.1.pb` -- the protocol buffer file (see [YOLO Processor](#yoloNet))
+     * `coco-1.1.meta` -- the associated meta file
+     * `coco-1.1.names` -- the (soon to be deprecated) label file
+ * Version 1.0
+     * `coco-1.0.pb` -- Same purpose as above for version 1.0
+     * `coco-1.0.names`
+* YOLO (deprecated) -- this is the same as COCO version 1.0, with less-well-chosen names
+  * `yolo.pb` -- the protocol buffer file -- the same contents as `coco-1.0.pb`
+  * `coco.names` -- the label file -- same contents as `coco-1.0.names`
+
+We have moved to new names as they are clearer about what they do.  Operationally, things will remain the same.
+
+You are free to use other YOLO-based models (or other neural nets if you provide your [own neural net processor via the `NeuralNetInterface`](#neuralNetInterface). While we have not tested with them, anything that uses a 416x416 image (that is, downsized to that before neural net processing) is likely to work.  We cannot test with all models. In the future, we may relax the 416x416 restriction by using information from the `.meta` file.  See the [YOLO Processor](#yoloNet) section for information on producing your own models.
+
+These files are now fetched from an Amazon S3-based maven repository, and will be downloaded into your gradle cache. For use in the tests, they are copied to `objectRecognitionSource/build/models`. If you want to use them for a project, you are encouraged to copy them to a stable location (the build hierarchy is destroyed when tests are run as part of normal gradle processing). If those files are missing, you can replace them by running the following tasks in gradle.
+
+* ```./gradlew :objectRecognitionSource:copyModels``` -- this will copy the current models (COCO v1.1) into ```build/models```
```

Comment:
Should we specify that these commands are for Macs, and that for Windows they should use an equivalent command to "./gradlew"?